apps/judicial-system/backend/migrations/20250206152023-update-case.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
Please do all these updates in a single transaction. Otherwise, the db may end up in an inconsistent state. Updating the enum cannot be done in a transaction, but the rest can. Updating the enum would just need to be done after all the other updates are completed.